eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -72,6 +75,25 @@ jobs:
             packagesToPush: '$(PIPELINE.WORKSPACE)/packages/**/*.nupkg;!$(PIPELINE.WORKSPACE)/packages/**/*.symbols.nupkg'
             nuGetFeedType: external
             publishFeedCredentials: Nuget.org
+      
+      - ${{ if eq(parameters.ShouldPublishToDevOps, 'true') }}:
+        - pwsh: |
+            # For safety default to publishing to the private feed.
+            # Publish to https://dev.azure.com/azure-sdk/internal/_packaging?_a=feed&feed=azure-sdk-for-net-pr
+            $devopsFeedId = '590cfd2a-581c-4dcb-a12e-6568ce786175/fa8b2d77-74d9-48d7-bb96-badb2b9c6ca4'
+            if ('$(Build.Repository.Name)' -eq 'Azure/azure-sdk-for-net') {
+              # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
+              $devopsFeedId = '${{ parameters.DevOpsFeedID }}'
```

Comment:
In this context I think we need to set hardcode this value as we don't have the parameter.

Reply:
Lets be sure that we try to test this as well.